docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -578,8 +578,10 @@ The following command operators are supported:
   is unique with high likelihood within an execution, but can duplicate across
   multiple instances or between restarts.
 
-%GRPC_STATUS%
-  gRPC status code which is easy to interpret with text message corresponding with number.
+%GRPC_STATUS(X)%
+  `gRPC status code <https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto>` in the optional parameter X. X can be ``CAMEL_STRING``, ``SNAKE_STRING`` and ``NUMBER``.
```

Comment:
wording. Maybe change it to 

gRPC status code <https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto>` formatted according to the optional parameter X, which can be ...

Reply:
Done. Thanks for the review!